include/deal.II/sundials/sundials_backport.h

```diff
@@ -53,10 +54,10 @@ namespace SUNDIALS
       ops->nvspace           = nullptr;
       ops->nvgetarraypointer = nullptr;
       ops->nvsetarraypointer = nullptr;
-#  if DEAL_II_SUNDIALS_VERSION_GTE(5, 0, 0)
+#    if DEAL_II_SUNDIALS_VERSION_GTE(5, 0, 0)
```

Comment:
I have deliberately kept all of these parts in place even though these preprocessor conditionals can never be true.